src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -180,12 +181,12 @@ public String getIdentificationString() {
 
     @Override
     public String getEntityTypeAsString() {
-        return "Feedback Session";
+        return ATTRIBUTE_NAME;
     }
 
     @Override
     public String getBackupIdentifier() {
-        return Const.SystemParams.COURSE_BACKUP_LOG_MSG + courseId;
+        return Const.SystemParams.BACKUP_LOG_MSG + ATTRIBUTE_NAME + "::" + feedbackSessionName;
```

Comment:
Same here, please investigate the code to find the unique combination.

e.g. Feedbacksession name + courseid